sources/packages/web/src/components/layouts/aest/AESTHomeSideBar.vue

```diff
@@ -74,92 +49,107 @@
 </template>
 <script lang="ts">
 import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
-import { MenuModel, Role } from "@/types";
+import { MenuItemModel, Role } from "@/types";
+import { ref } from "vue";
 import CheckPermissionRole from "@/components/generic/CheckPermissionRole.vue";
 
 export default {
   components: { CheckPermissionRole },
   setup() {
-    const topItems = [
+    const topItems = ref<MenuItemModel[]>([
       {
-        label: "Home",
-        icon: "mdi-home-outline",
-        command: () => ({
-          name: AESTRoutesConst.AEST_DASHBOARD,
-        }),
+        title: "Home",
+        props: {
+          prependIcon: "mdi-home-outline",
+          to: {
+            name: AESTRoutesConst.AEST_DASHBOARD,
+          },
+        },
       },
       {
-        label: "Search Students",
-        icon: "mdi-magnify",
-        command: () => ({
-          name: AESTRoutesConst.SEARCH_STUDENTS,
-        }),
+        title: "Search Students",
+        props: {
+          prependIcon: "mdi-magnify",
+          to: {
+            name: AESTRoutesConst.SEARCH_STUDENTS,
+          },
+        },
       },
       {
-        label: "Search Institutions",
-        icon: "mdi-magnify",
-        command: () => ({
-          name: AESTRoutesConst.SEARCH_INSTITUTIONS,
-        }),
+        title: "Search Institutions",
+        props: {
+          prependIcon: "mdi-magnify",
+          to: {
+            name: AESTRoutesConst.SEARCH_INSTITUTIONS,
+          },
+        },
       },
-    ];
-
-    const studentAccountApplicationItem = {
-      label: "Accounts",
-      icon: "mdi-account-outline",
-      command: () => ({
-        name: AESTRoutesConst.STUDENT_ACCOUNT_APPLICATIONS,
-      }),
-    } as MenuModel;
-
-    const exceptionsItem = {
-      label: "Exceptions",
-      icon: "mdi-alert-circle-outline",
-      command: () => ({
-        name: AESTRoutesConst.APPLICATION_EXCEPTIONS_PENDING,
-      }),
-    } as MenuModel;
-
-    const appealsItem = {
-      label: "Appeals",
-      icon: "mdi-folder-open-outline",
-      command: () => ({
-        name: AESTRoutesConst.APPLICATION_APPEALS_PENDING,
-      }),
-    } as MenuModel;
-
-    const designations = {
-      label: "Pending designations",
-      icon: "mdi-bookmark-outline",
-      command: () => ({
-        name: AESTRoutesConst.PENDING_DESIGNATIONS,
-      }),
-    } as MenuModel;
-
-    const reports = {
-      label: "Reports",
-      icon: "mdi-home-outline",
-      command: () => ({
-        name: AESTRoutesConst.REPORTS,
-      }),
-    } as MenuModel;
-
-    const offerings = {
-      label: "Offerings",
-      icon: "mdi-view-list-outline",
-      command: () => ({
-        name: AESTRoutesConst.OFFERING_CHANGE_REQUESTS,
-      }),
-    } as MenuModel;
-
+    ]);
+    const studentRequestsItems = ref<MenuItemModel[]>([
```

Comment:
you have everything in place just add  `studentRequestsItems` and `institutionRequestsItems` to `topItems`.